src/AppBundle/Utils/Markdown.php

```diff
@@ -25,7 +25,10 @@ class Markdown
 
     public function __construct()
     {
-        $this->parser = new \Parsedown();
+        $parsedown = new \Parsedown();
```

Comment:
it's not related to original bug but it's looking weird to have `new` in construct. maybe we can define `Parsedown` it as service and use DI to pass it here as a constructor parameter?